core/src/main/java/brooklyn/catalog/internal/BasicBrooklynCatalog.java

```diff
@@ -186,12 +192,7 @@ public CatalogDo getCatalog() {
             return null;
         }
 
-        String versionedId = CatalogUtils.getVersionedId(symbolicName, fixedVersionId);
-        CatalogItemDo<?, ?> item = null;
-        //TODO should remove "manual additions" bucket; just have one map a la osgi
-        if (manualAdditionsCatalog!=null) item = manualAdditionsCatalog.getIdCache().get(versionedId);
-        if (item == null) item = catalog.getIdCache().get(versionedId);
-        return item;
+        return catalog.getIdCache().get( CatalogUtils.getVersionedId(symbolicName, fixedVersionId) );
```

Comment:
I don't follow the use of `manualAdditionsCatalog`. Can we safely ignore it in this method?

I see manualAdditionsCatalog is used in the deprecated `addItem(CatalogItem<?,?> item)`, but `catalog` is not.